test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -11218,12 +11061,16 @@ int main(int argc, char** argv) {
   // Workaround Apple CFStream bug
   gpr_setenv("grpc_cfstream", "0");
 #endif
+  grpc::testing::g_fake1_cert_data_map =
+      new grpc::testing::FakeCertificateProvider::CertDataMapWrapper();
```

Comment:
No need for dynamic allocation here, since you're deleting the objects before this scope ends.  Just declare a local variable and set the global pointer to point to it.